src/actions/users.test.js

```diff
@@ -0,0 +1,70 @@
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { USER_INFO_REQUEST, USER_INFO_SUCCESS, USER_INFO_FAILURE, fetchUserInfo } from './users';
+import { ERROR } from './errors';
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+
+const userInfoRequest = 'USER_INFO_REQUEST';
+const userInfoSuccess = 'USER_INFO_SUCCESS';
+const userInfoFailure = 'USER_INFO_FAILURE';
+
+const result = {
+  accessToken: 'vlseTWA8LXFz6X5kBH2BJO2l33GRZSPZAHA2-FHGC94',
+  name: 'mockuser',
+  email: 'mockuser@example.com'
+};
+
+describe('All constants remain unchanged', () => {
+  it('USER_INFO_REQUEST == USER_INFO_REQUEST', () => {
+    expect(USER_INFO_REQUEST).toEqual(userInfoRequest);
+  });
+
+  it('USER_INFO_SUCCESS == USER_INFO_SUCCESS', () => {
+    expect(USER_INFO_SUCCESS).toEqual(userInfoSuccess);
+  });
+
+  it('USER_INFO_FAILURE == USER_INFO_FAILURE', () => {
+    expect(USER_INFO_FAILURE).toEqual(userInfoFailure);
+  });
+});
+
+describe('Test actions', () => {
+  let store;
+  let initialState;
+
+  beforeAll(() => {
+    initialState = {};
+  });
+
+  beforeEach(() => {
+    store = mockStore(initialState);
+  });
+
+  describe('fetchUserInfo', () => {
+    it('successfully fetch user info', () => {
```

Comment:
As we are mainly testing that the correct actions are created here I think the description should focus on the actions we expect. So for this one it could be:

```suggestion
it('creates USER_INFO_SUCCESS action', () => {}
```